Polish the doc of v1.0.3 release (#190)
harvester/docs: docs/monitoring/monitoring.md

    $ free -h
                  total        used        free      shared  buff/cache   available
    Mem:          7.7Gi       166Mi       4.6Gi       1.0Mi       2.9Gi       7.2Gi
    Swap:            0B          0B          0B
    ```

    The corresponding `Memory Usage` is `(1 - 4.6/7.7) * 100%`, roughly `40%`.

## How to Config Monitoring Settings

_Available as of v1.0.1_

Monitoring has several components that help to collect and aggregate metric data from all Nodes/Pods/VMs. The resources required for monitoring are depends on your workloads and hardware resources. Harvester sets defaults based on general use case, and you can change them accordingly.

Currently, `Resources Settings` can be configured for the following components:

- Prometheus
- Prometheus Node Exporter(_UI configurable as of v1.0.2_)

### From WebUI

In the `Advanced Settings` page, you can view and change the resources settings as follows:

1. Navigate to settings page, find `harvester-monitoring`.
![](./assets/monitoring-setting.png)

1. Click `Show harvester-monitoring` to view the current values.

1. Click `⋮ > Edit Setting` to set a new value.
![](./assets/monitoring-setting-edit-config.png)

> 1. Click `Save` and the Monitoring will be restarted within a few seconds. Please be aware that the reboot can take some time to reload previous data.

Reviewer: ```suggestion
1. Click `Save` and the `Monitoring` resource will be restarted within a few seconds. Please be aware that the reboot can take some time to reload previous data.
```

**The most frequently used option is the memory setting:**

> - The `Requested Memory` is the minimum memory of the `Monitoring`. The recommended value is about 5% to 10% of the system memory of one single management node. A value less than 500Mi will be denied.

Reviewer: ```suggestion
- The `Requested Memory` is the minimum memory required by the `Monitoring` resource. The recommended value is about 5% to 10% of the system memory of one single management node. A value less than 500Mi will be denied.
```

> - The `Memory Limit` is the maximum memory of the `Monitoring`. The recommended value is about 30% of system memory of one single management node, when the `Monitoring` reaches this threshold, it will automatically restart.

Reviewer: ```suggestion
- The `Memory Limit` is the maximum memory that can be allocated to a `Monitoring` resource. The recommended value is about 30% of the system's memory for one single management node. When the `Monitoring` reaches this threshold, it will automatically restart.
```

Depending on the available hardware resources and system loads, you may change the above settings accordingly.

!!! note
    If you have multiple management nodes with different hardware resources, please set the value of Prometheus based on the smaller one.

!!! attention

>     When more and more VMs get deployed in one node, the `prometheus-node-exporter` pod might get killed due to OOM(out of memory). In that case, you should increase the value of `limits.memory`.

Reviewer: ```suggestion
    When an increasing number of VMs get deployed on one node, the `prometheus-node-exporter` pod might get killed due to OOM(out of memory). In that case, you should increase the value of `limits.memory`.
```
